src/Spectre.libGenetic/CrossoverOperator.h

```diff
@@ -33,19 +34,43 @@ class CrossoverOperator
     /// Initializes a new instance of the <see cref="CrossoverOperator"/> class.
     /// </summary>
     /// <param name="rngSeed">The RNG seed.</param>
-    explicit CrossoverOperator(Seed rngSeed = 0, size_t minimalFillup=0, size_t maximalFillup=std::numeric_limits<size_t>::max());
+    /// <param name="minimalFillup">The maximal fillup.</param>
+    /// <param name="maximalFillup">The minimal fillup.</param>
+    /// <param name="individualFeasibilityCondition">The individual feasibility condition.</param>
+    explicit CrossoverOperator(Seed rngSeed = 0, size_t minimalFillup = 0, size_t maximalFillup = std::numeric_limits<size_t>::max(),
+                                BaseIndividualFeasibilityCondition* individualFeasibilityCondition = nullptr);
     virtual ~CrossoverOperator() = default;
     /// <summary>
+    /// Create new individual until it its conditions.
+    /// </summary>
+    /// <param name="first">The first parent.</param>
+    /// <param name="second">The second parent.</param>
+    /// <returns>A child fulfilling conditions.</returns>
+    virtual Individual operator()(const Individual &first, const Individual &second);
+    /// <summary>
     /// Create new individual.
     /// </summary>
     /// <param name="first">The first parent.</param>
     /// <param name="second">The second parent.</param>
     /// <returns>A child.</returns>
-    virtual Individual operator()(const Individual &first, const Individual &second);
+    virtual Individual cross(const Individual &first, const Individual &second);
```

Comment:
If we have two options: `operator()` and `cross`, it will be hard to know which one to use at a glance. Either hide `cross`, or figure out something else for making this straightforward.

Reply:
done